build/bin/upload

```diff
@@ -26,12 +26,17 @@ then
 fi
 
 echo "Archiving files from $must_gather_output to $must_gather_upload/${CSP_FILENAME}"
-tar caf "$must_gather_upload/${CSP_FILENAME}" $must_gather_output/
+tar --ignore-failed-read -caf "$must_gather_upload/${CSP_FILENAME}" $must_gather_output/
```

Comment:
@dustman9000 this PR adds a few new features that Benson and I uncovered as we tried to fix the issue that was happening in OSD-8230. that issue in particular was due to not having root permissions in the file system that would allow the cluster-logging must-gather to tar up some files like elasticsearch keys etc. we decided CEE probably didn't need access to those keys, and that it would be safe to just not tar those up. this particular change keeps the tar command from failing out when it tries to tar up those files